packages/nimbus/src/components/date-range-picker/date-range-picker.dev.mdx

```diff
@@ -0,0 +1,235 @@
+  </Stack>
+)
+```
+
+#### Variant Options
+
+```jsx-live-dev
+const App = () => (
+  <Stack direction="column" gap="400">
+    <DateRangePicker variant="solid" width="8000" />
+    <DateRangePicker variant="ghost" width="8000" />
+  </Stack>
+)
+```
+
+#### Granularity Options
+
+```jsx-live-dev
+const App = () => (
+  <Stack direction="column" gap="400">
+    <DateRangePicker granularity="day" width="8000" />
+    <DateRangePicker granularity="minute" width="10000" />
+  </Stack>
+)
+```
+
+#### Controlled State
+
+```jsx-live-dev
+const App = () => {
+  const [value, setValue] = useState(null);
+  
+  return (
+    <Stack direction="column" gap="400">
+      <DateRangePicker 
+        value={value} 
+        onChange={setValue} 
+        width="8000" 
+      />
+      <Text fontSize="sm">
+        Selected: {value ? `${value.start} to ${value.end}` : 'None'}
+      </Text>
+    </Stack>
+  );
+}
+```
+
+## Testing Strategies and Best Practices
+
+The DateRangePicker component requires comprehensive testing across multiple interaction patterns.
+
+#### 1. Unit Testing with React Testing Library
+
+```tsx
+import { render, screen } from '@testing-library/react';
+import { DateRangePicker } from '@commercetools/nimbus';
+
+describe('DateRangePicker', () => {
+  it('renders start and end date inputs', () => {
+    render(<DateRangePicker />);
+    const dateInputs = screen.getAllByRole('spinbutton');
+    expect(dateInputs).toHaveLength(6); // 3 segments per date input
+  });
+
+  it('shows calendar and clear buttons', () => {
+    render(<DateRangePicker />);
+    expect(screen.getByRole('button', { name: /open calendar/i })).toBeInTheDocument();
+    expect(screen.getByRole('button', { name: /clear selection/i })).toBeInTheDocument();
+  });
+});
+```
+
+#### 2. Storybook Integration for Testing
+
+Leverage existing stories for comprehensive visual and interaction testing:
+
+```tsx
+export const InteractionTest: Story = {
+  play: async ({ canvasElement }) => {
+    const canvas = within(canvasElement);
+    
+    // Test calendar opening
+    const calendarButton = await canvas.findByRole('button', { name: /calendar/i });
+    await userEvent.click(calendarButton);
+    
+    // Verify calendar popover appears
+    await waitFor(() => {
+      const calendar = within(document.body).queryByRole('application');
+      expect(calendar).toBeInTheDocument();
+    });
+    
+    // Test date range selection in calendar
+    const dates = within(document.body).getAllByRole('gridcell');
+    await userEvent.click(dates[10]); // Start date
+    await userEvent.click(dates[15]); // End date
+    
+    // Verify selection reflected in inputs
+    const segments = canvas.getAllByRole('spinbutton');
+    expect(segments[0]).toHaveValue('10');
+  },
+};
+```
+
+#### 3. Regression tests
+
+[Placeholder for regression test strategy]
+
+Visual regression with Chromatic
+Behavior regression with interaction tests
+API contract testing
+
```

Comment:
I don't think these two sections are necessary for initial documentation, we don't have VRT's in place, and storybook isnt widely used in the MC, etc